### DISCUSSION TITLE
Are challenges' solutions available somewhere?

### DISCUSSION THREAD

**Discussion by JJScar** [tinchoabbate/damn-vulnerable-defi#56] [Help / Q&A / Troubleshooting] [answered]
Hello! 
I just finished Patrick Collins's part 1 of the security course. I wanted to sharpen up my skills by coming here. I just want to know about what to do about the answers.
I just did the `Unstoppable` challenge and got the test to pass, however it literally took me one line of code and I feel like there are more ways.  Is there somewhere we can look for solved challenges to compare our work, or if we are completely stuck? Because I could not find anything. 

Thanks for the help!

**Comment by JJScar** [comment]
Also to add! I would love to see the prevention the should be implemented. Again I will do everything I can to try myself, but to learn I must first try and then compare my work to someone like you @tinchoabbate

**Reply by tinchoabbate** [reply]
Same answer as below, I'd rather you come up with your own! In your writeups you can include your approach to trying to fix the vulnerabilities you found. A big part of Damn Vulnerable DeFi is just going through the learning process, regardless of how accurate your answers are.

**Comment by tinchoabbate** [comment, marked as answer]
Hey @JJScar, thanks for the questions!

For the solutions, some will take you literally one line of code, others much more. Depends on the challenge! And I'm pretty sure there's no "right" answer. As long as the test passes, you should be good! What I'd suggest you do is publish your writeups as you solve each challenge (in your own blog, medium, etc), and share them with the community. I'm sure people will be interested in learning from your approach.

I do have my own solutions, which I haven't published yet. I'd rather have people come up with their own and not influence anyone!

**Reply by JJScar** [reply]
I'm all for the challenge of course! However, I'm very interested to see how you went about it! Appreciate the response and sure I will share my approach to everyone to help others and to hear criticism. 
Thanks for the response!
